src/test/kotlin/step4/domain/ForwardTest.kt

```diff
@@ -0,0 +1,23 @@
+package step4.domain
+
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+
+/**
+ *
+ * @author Leo
+ */
+class ForwardTest {
+
+    @Test
+    fun `100퍼센트 확률일시 전진 되는지 검증`() {
+        // given
+        var forward: Forward
+
+        // when
+        forward = FakeForwardImpl()
+
+        // then
+        assertTrue(forward.moved())
```

Comment:
이 부분도 assertJ를 이용해서
```suggestion
        // then
        assertThat(forward.moved()).isTrue()
```
이렇게 사용가능합니다